xds/src/main/java/io/grpc/xds/PriorityLoadBalancer.java

```diff
@@ -153,7 +153,9 @@ private void tryNextPriority(boolean reportConnecting) {
     }
     // TODO(zdapeng): Include error details of each priority.
```

Comment:
We could delete this TODO. It's a bit undefined for how to aggregate multiple error statuses.

Reply:
I was considering a status with a cause that [suppresses](https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html#getSuppressed()) a list of status exceptions for all priorities, or something similar, as further improvement in the future.

Reply:
That sounds a horrible idea though. Using the last priority's connection status is good enough as per current design.